test/unit/core.js

```diff
@@ -1709,3 +1709,45 @@ QUnit.test( "Iterability of jQuery objects (gh-1693)", function( assert ) {
 		assert.ok( true, "The browser doesn't support Symbols" );
 	}
 } );
+
+QUnit[ jQuery.Deferred ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
```

Comment:
Do we have plan for this? If we do, maybe we should put it on roadmap or create ticket for this?